src/org/usfirst/frc/team3407/robot/commands/OpenArms.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc.team3407.robot.commands;
+
+import org.usfirst.frc.team3407.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class OpenArms extends Command {
+
+    public OpenArms() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	requires(Robot.arms);	
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+    
+    // Called repeatedly when this Command is scheduled to run
+    
+    //Could possibly not work? Thread.sleep is funky.
+    protected void execute() {
+    	Robot.arms.open();
```

Comment:
This may need to change once we see how the arms actually open.  The motor may need to run while the button is pressed instead of a fixed amount of time.